privacyidea/lib/policydecorators.py

```diff
@@ -300,8 +300,8 @@ def auth_user_passthru(wrapped_function, user_object, passw, options=None):
                                                user=user_object.login,
                                                client=clientip, active=True)
         if len(pass_thru) > 1:
-            log.debug(u"Contradicting passthru policies: {0!s}".format(pass_thru))
-            raise PolicyError("Contradicting passthru policies.")
+            pass_thru = policy_object.choose_prioritized_policy(pass_thru)
+            log.debug(u"Multiple passthru policies, choosing {0!s}".format(pass_thru[0]))
```

Comment:
Should we add this logic to ``get_policies``?

As we now have a solution for contradiction policies, there should be no contradicting policies anymore. So we could move the remaining contradicting stuff the the lower level of get_policies.

Depending on additional parameters like *sorted* and *unique* we could

1. return a sorted list, like for displaying in the webui.
2. return the top most priority policy if *unique* is set. And only if the logic is not clear, raise an exception in case of a failing *unique* from within ``get_policies``.